core/src/main/java/org/infinispan/util/rxjava/FlowableFromIntSetFunction.java

```diff
@@ -0,0 +1,424 @@
+package org.infinispan.util.rxjava;
+
+import java.util.PrimitiveIterator;
+import java.util.function.IntFunction;
+
+import org.infinispan.commons.util.IntSet;
+import org.reactivestreams.Subscriber;
+
+import io.reactivex.Flowable;
+import io.reactivex.annotations.Nullable;
+import io.reactivex.exceptions.Exceptions;
+import io.reactivex.internal.functions.ObjectHelper;
+import io.reactivex.internal.fuseable.ConditionalSubscriber;
+import io.reactivex.internal.subscriptions.BasicQueueSubscription;
+import io.reactivex.internal.subscriptions.EmptySubscription;
+import io.reactivex.internal.subscriptions.SubscriptionHelper;
+import io.reactivex.internal.util.BackpressureHelper;
+
+/**
+ * TODO: add docs
```

Comment:
Don't forget :smiley:

Reply:
Yep, I still have to go through and do all of them.